### PR TITLE
Update Changelog - Prepare Release

### PR DESCRIPTION
**Pull request by gzussa**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,12 @@
 # CHANGELOG
 
+## v1.0.0-beta.4 / 2020-06-09
+
+* [Added] Generate API for Logs Archives. See [#323](https://github.com/DataDog/datadog-api-client-go/pull/323).
+* [Changed] Fix enums. See [#320](https://github.com/DataDog/datadog-api-client-go/pull/320).
```
Comment (zippolyte) [inline CHANGELOG.md:6]: can we have a better description here ?

Comment (gzussa) [inline CHANGELOG.md:6]:
```suggestion
* [Changed] Adding missing values to enums. See [#320](https://github.com/DataDog/datadog-api-client-go/pull/320).
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,12 @@
 # CHANGELOG
 
+## v1.0.0-beta.4 / 2020-06-09
+
+* [Added] Generate API for Logs Archives. See [#323](https://github.com/DataDog/datadog-api-client-go/pull/323).
```
Comment (gzussa) [inline CHANGELOG.md:5]:
```suggestion
* [Added] Add Logs Archives endpoints. See [#323](https://github.com/DataDog/datadog-api-client-go/pull/323).
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,12 @@
...
+* [Changed] Add missing values to enums. See [#320](https://github.com/DataDog/datadog-api-client-go/pull/320).
```
Comment (gzussa) [inline CHANGELOG.md:6]:
```suggestion
* [Changed] Add missing values to enums. See [#320](https://github.com/DataDog/datadog-api-client-go/pull/320).
    * This change includes backwards incompatible changes when using the `MonitorSummary` widget.
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,12 @@
...
+* [Changed] Split schemas from DashboardList v2. See [#318](https://github.com/DataDog/datadog-api-client-go/pull/318).
```
Comment (gzussa) [inline CHANGELOG.md:7]:
```suggestion
* [Changed] Split schemas from DashboardList v2. See [#318](https://github.com/DataDog/datadog-api-client-go/pull/318).
    * This change includes backwards incompatible changes when using corresponding endpoints methods
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,14 @@
 # CHANGELOG
 
+## v1.0.0-beta.4 / 2020-06-09
+
+* [Added] Add Logs Archives endpoints. See [#323](https://github.com/DataDog/datadog-api-client-go/pull/323).
+* [Changed] Add missing values to enums. See [#320](https://github.com/DataDog/datadog-api-client-go/pull/320).
+    * This change includes backwards incompatible changes when using the `MonitorSummary` widget.
+* [Changed] Split schemas from DashboardList v2. See [#318](https://github.com/DataDog/datadog-api-client-go/pull/318).
+    * This change includes backwards incompatible changes when using corresponding endpoints methods
```
Comment (zippolyte) [inline CHANGELOG.md:9]:
```suggestion
    * This change includes backwards incompatible changes when using corresponding endpoints methods.
```

**Pull request merged**